composer.json

```diff
@@ -1,32 +1,45 @@
 {
-  "name": "cwp/cwp-installer",
-  "description": "CWP Project Template",
-  "license": "BSD-3-Clause",
-  "require": {
-    "cwp/cwp-recipe-basic": "1.8.x-dev",
-    "cwp/cwp-recipe-blog": "1.8.x-dev",
-    "cwp/starter-theme": "1.1.x-dev"
-  },
-  "require-dev": {
-    "cwp/cwp-recipe-basic-dev": "^1.1@stable"
-  },
-  "suggest": {
-    "silverstripe/textextraction": "Required for document text searching within CWP"
-  },
-  "config": {
-    "process-timeout": 900
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.8.x-dev"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.cwp.govt.nz/"
-    }
-  ],
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "cwp/cwp-installer",
+    "type": "silverstripe-recipe",
+    "description": "CWP Project Template",
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": ">=5.6.0",
+        "silverstripe/recipe-plugin": "^1",
+
+        "silverstripe/recipe-cms": "1.1.x-dev",
+
+        "cwp/cwp-recipe-core": "2.x-dev",
+
+        "cwp/cwp": "2.x-dev",
+        "silverstripe/html5": "2.x-dev",
+        "symbiote/silverstripe-gridfieldextensions": "3.0.x-dev",
+        "cwp/cwp-pdfexport": "1.x-dev",
+
+        "cwp/cwp-search": "1.x-dev",
+        "symbiote/silverstripe-queuedjobs": "4.x-dev",
+        "silverstripe/textextraction": "3.x-dev",
+
+        "silverstripe/subsites": "2.0.x-dev",
+        "tractorcow/silverstripe-fluent": "4.0.x-dev",
+        "silverstripe/registry": "2.x-dev",
+        "cwp/starter-theme": "2.x-dev"
```

Comment:
This should have all the recipes too. E.g. `silverstripe/recipe-blog`.

Reply:
Good reminder. `silverstripe/recipe-blog` wasn't (and still isn't) a thing yet though ;)
https://github.com/silverstripe/recipe-blog/pull/1

Reply:
This is now done.